app/build_svc.py

```diff
@@ -164,24 +171,24 @@ def _stage_payload(self, language, payload):
         if os.path.exists(src):
             shutil.move(src=src, dst=dst)
 
-    async def _stage_docker_directory(self, env, ability):
+    async def _stage_docker_directory(self, env, executor):
         """Create code file and references in Docker build directory
 
         :param env: Build environment settings
         :type env: dict
-        :param ability: Ability to be built
-        :type ability: Ability
+        :param executor: Executor to be built
+        :type executor: Executor
         """
         build_file = '{}.{}'.format(self.build_file, env['extension']) if env.get('extension') else self.build_file
-        with open(os.path.join(self.build_directory, ability.language, build_file), 'w') as f:
-            f.write(self.decode_bytes(ability.code, strip_newlines=False))
+        with open(os.path.join(self.build_directory, executor.language, build_file), 'w') as f:
+            f.write(executor.code)
```

Comment:
Previously this was calling `decode_bytes()` and also passed in a `strip_newlines=False`. Was it intentional to leave those off?

Reply:
`code` is no longer base64 encoded, so `decode_bytes()` is unnecessary. `strip_newlines=False` was added because the builder languages care about newlines.